wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireExtension.kt

```diff
@@ -277,16 +295,19 @@ open class WireExtension(
     internal val includes = mutableListOf<String>()
     internal val excludes = mutableListOf<String>()
 
+    /** Sets a directory. Example: "src/main/proto". */
     fun srcDir(dir: String) {
       isEmpty = false
       sourceDirectoriesAndLocalJars += project.file(dir)
     }
 
+    /** Sets one or more directory. */
```

Comment:
Shouldn't it be plural?

```suggestion
    /** Sets one or more directories. */
```